.evergreen/secrets_handling/README.md

```diff
@@ -36,11 +36,11 @@ If using a Linux host on Evergreen, the shorthand version of the script can be u
 
 ```yaml
 - command: subprocess.exec
-    params:
+  params:
     working_dir: src
     binary: bash
     args: |
-        ${DRIVERS_TOOLS}/.evergreen/atlas/setup-secrets.sh
+      ${DRIVERS_TOOLS}/.evergreen/atlas/setup-secrets.sh
```

Comment:
I'm not convinced that the `args: |` line is correct.

`evergreen validate` reports:

```
ERROR: tasks section in 'subprocess.exec' command: parsing parameters for command 'subprocess.exec' (step 1 of 2): decoding mapstructure params: 1 error(s) decoding:

* 'args': source data must be an array or slice, got string
```

The docs for [subprocess.exec](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands/#subprocessexec) states that `args` should be a list.

Reply:
Yeah, it should look like:

```
args:
          - ${DRIVERS_TOOLS}/...
```

I missed that when converting the examples from `shell.exec`.

Reply:
Updated.